packages/docs/docs/about.mdx

```diff
@@ -64,26 +92,3 @@ function CharacterGenerator() {
 ```
 
 [![AI.JSX demo](https://asciinema.org/a/m08ORXH8AdcmV8gA95d6UYJzW.svg)](https://asciinema.org/a/m08ORXH8AdcmV8gA95d6UYJzW)
```

Comment:
This looks great – if we could embed the video player directly in the page, that would be even better. (But this doesn't have to be a huge priority.)